### PR TITLE
Making minor changes from review.

### PR DESCRIPTION
**Pull request by DesertBeagle**

### CODE REVIEW

**Comment by DesertBeagle** [comment]
@Sir-Batman @bitschift What could be added?

File: docs/problem_statement.tex [hunk 1]
```diff
@@ -51,7 +51,7 @@ \section{Proposed Solution}
 
 \subsection{Hardware}
 In order to create a solid and reliable automated home-brewing setup, our hardware 
-implementation needs to be simple and modular to accommodate peoples' various setups.
+implementation needs to be simple and modular to accommodate peoples' various brewing arrangements.
```
Comment (Sir-Batman) [inline docs/problem_statement.tex]: This change doesn't make "setups" much clearer. It should be a larger expansion of the term. But potentially more important, this sentence doesn't say much of anything, and what it does say ("The final product needs to be modular and minimal for integration into existing brewing arrangements") is way too wordy. It is unclear in this sentence what "hardware" we are talking about - is it the electrical hardware we are using, or the brewing setup? It seems to be about the brewing setup, but we also talk about the "hardware" being the electrical stuff in other discussions. We need to be consistent on that front.

**Comment by Sir-Batman** [comment]
Looks good.
